.github/workflows/code_quality.yml

```
@@ -0,0 +1,56 @@
+name: Code quality checks
+
+# Run this workflow every time a new commit pushed to your repository
+on:
+  push:
+    branches:
+      - master
+      - stable/*
+    tags:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
+  pull_request:
+    paths:
+      - '**.py'
+      - '**.json'
+      - '**.yaml'
+      - '**.in'
```

I think we can simplify this, given the small size of the library and test suite?

```suggestion
  push:
    branches:
      - master
    tags:
      - '*'
  pull_request:
```